onnx_coreml/_backend.py

```diff
@@ -20,7 +40,8 @@ def prepare(cls,
         # type: (...) -> CoreMLRep
         super(CoreMLBackend, cls).prepare(model, device, **kwargs)
         coreml_model = convert(model)
-        return CoreMLRep(coreml_model, device == 'CPU')
+        onnx_outputs = _get_onnx_outputs(model) # type: ignore
```

Comment:
You won't need `type: ignore` if `_get_onnx_outputs` has a type hint

Reply:
I have removed the ignore.